group-anagrams/HodaeSsi.py

```diff
@@ -0,0 +1,10 @@
+# 시간복잡도 : O(n*mlogm) (n은 strs의 길이, m은 strs의 원소의 길이)
+# 공간복잡도 : O(n)
+
+class Solution:
+	def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
+		anagrams = collections.defaultdict(list)
+		for word in strs:
+			anagrams[''.join(sorted(word))].append(word)
```

Comment:
```suggestion
                        key = ''.join(sorted(word))
			anagrams[key].append(word)
```

코딩 골프 느낌의 pythonic한 짧은 코드도 좋지만, 해당 임시 변수가 어떤 역할을 하는지 선언한 변수명으로 알려주면 좋을 것 같습니다.